packages/framework/.travis.yml

```
@@ -10,6 +10,7 @@ cache:
         - ~/.composer/cache
 
 before_install:
+    - echo '' | pecl upgrade redis
```

```suggestion
    - echo '' | pecl install redis-4.1.1
```
so it's consistent with https://github.com/shopsys/shopsys/blob/v7.2.0/.travis.yml#L16

---

it because of travis php 7.2,7.3 images. there is php-redis module in higher version and pecl fails with error